docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,134 @@
+# Contributing
+
+The `Substrate` project is an ***OPENISH Open Source Project***
+
+Contributors are invited to our `#frame-contributors` channel on the Polkadot Discord for support and coordination:  
+[![Discord](https://img.shields.io/discord/722223075629727774?style=for-the-badge&logo=discord&label=Discord)](https://dot.li/discord)
```

Comment:
This was added

Reply:
Not sure why we bring back a channel that we deprecated a year ago and then we also do this on discord...

Reply:
Its for making the external FRAME contributors feel more included for getting hints about their issues and coordination.  
All technical Qs will still be redirected to SE etc.